src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+
+        // Text declarations and initial greeting
+        final String BARRIER = "____________________________________________________________";
+        final String LOGO = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        final String GREETING = BARRIER + "\n\nAhoy there! I be\n" + LOGO + "\nWhat can I do for ye?\nFor a list of commands, type 'help' me hearties!\n" + BARRIER + "\n";
```

Comment:
Line length should be no longer than 120 chars. Use line wrapping at the appropriate place on the sentence